check_domain_expiration.sh

```diff
@@ -234,6 +234,16 @@ check_domain()
 		EXDATE_TMP=$(${WHOIS} -h whois.nic.im "${1}" | ${AWK} '/Expiry Date:/ { gsub("[:.]","-"); print $3 }' | cut -d 'T' -f1 | awk -F[/] '{print $2"/"$1"/"$3}') 
 		EXDATE=`date -d"$EXDATE_TMP" +%Y-%m-%d`
 		EXP_DAYS=$(( ( $(date -ud ${EXDATE} +'%s') - $(date -ud `date +%Y-%m-%d` +'%s') )/60/60/24 ))
+        elif [ "$DTYPE" == "coop" ]
+        then
+                EXDATE_TMP=$(${WHOIS} -h whois.nic.coop "${1}" | ${AWK} '/Expiry Date:/ { print $4 }' | cut -c 1-16)
+                EXDATE=`date -d"$EXDATE_TMP" +%Y-%m-%d`
```

Comment:
It should be done in accordance with #15 – it fixed the problem with no expiration date in the WHOIS response. Could you please update the code to meet these requirements? As an example, you can use `.moscow` domain. I mean additional conditional that checks if `EXDATE` is empty:

```
EXDATE_TMP=$(${WHOIS} "${1}" | ${AWK} '/Registry Expiry Date:/ { print $4 }' | cut -c 1-16)
if [ -z "$EXDATE_TMP" ]
then
    EXP_DAYS=NULL
```